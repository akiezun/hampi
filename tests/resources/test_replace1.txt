S = X "a" ;
X = S "x" ;