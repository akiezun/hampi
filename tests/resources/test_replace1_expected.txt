S = S "a"  | S "x"  ;
